sockeye/data_io.py

```diff
@@ -1333,10 +1349,11 @@ def __init__(self,
                  target_data_name=C.TARGET_NAME,
                  label_name=C.TARGET_LABEL_NAME,
                  num_factors: int = 1,
+                 shuffle: Optional[bool] = True,
```

Comment:
`Optional` means it can be `None`, I guess that's not intended here. Use `bool` instead.